be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/MeetingValidator.scala

```diff
@@ -70,24 +86,52 @@ sealed class MeetingValidator(dbActorRef: => AskableActorRef) extends MessageDat
         val laoId: Hash = rpcMessage.extractLaoId
```

Comment:
You can use the function `extractData` from the `MessageValidator.scala` file. Thus, you can make your code more concise.

Reply:
I reiterate with this comment. Try to use `extractData`.